test/PoolManagerInitialize.t.sol

```diff
@@ -49,7 +49,7 @@ contract PoolManagerInitializeTest is Test, Deployers, GasSnapshot {
         });
     }
 
-    function test_initialize(PoolKey memory key0, uint160 sqrtPriceX96) public {
+    function test_initialize_fuzz(PoolKey memory key0, uint160 sqrtPriceX96) public {
```

Comment:
test_fuzz_initialize just so that we can do things like `forge test --mt test_fuzz `to run all fuzz